src/main/java/uk/co/fivium/els/categories/tumbledryers/model/AirVentedTumbleDryersForm.java

```diff
@@ -0,0 +1,94 @@
+package uk.co.fivium.els.categories.tumbledryers.model;
+
+import javax.validation.constraints.Digits;
+import javax.validation.constraints.NotBlank;
+import uk.co.fivium.els.model.FieldPrompt;
+
+public class AirVentedTumbleDryersForm {
+
+  @FieldPrompt("Supplier's name or trade mark")
+  @NotBlank(message = "Enter a supplier name or trade mark") // TODO max length?
+  private String supplierName;
+
+  @FieldPrompt("Supplier's model identifier")
+  @NotBlank(message = "Enter a supplier model identifier")
+  private String modelName;
+
+  @FieldPrompt("Energy efficiency class of the tumble dryer")
+  @NotBlank(message = "Select an energy efficiency class")
+  private String efficiencyRating;
+
+  @FieldPrompt("Weighted annual energy consumption (AEC) in kWh per year, rounded up to the nearest integer")
+  @Digits(integer = 3, fraction = 0, message = "Enter an energy consumption, up to 3 digits long")
+  private String energyConsumption;
+
+  @FieldPrompt("Cycle time corresponding to the standard cotton programme at full load in minutes per cycle")
+  @Digits(integer = 3, fraction = 0, message = "Enter the cycle time in minutes, up to 3 digits long")
+  private String cycleTime;
+
+  @FieldPrompt("Rated capacity, for the standard cotton programme at full load in kg")
+  @Digits(integer = 1, fraction = 10, message = "Enter the capacity, rounded to the nearest 0.1 kg")
```

Comment:
I'd also change this message to 'Enter the capacity, as 1 digit and with an optional decimal place'. Currently someone might enter 10 and it might not be clear why it was invalid from the message.